src/PostgREST/Error.hs

```diff
@@ -171,7 +171,7 @@ instance JSON.ToJSON ApiRequestError where
     "code"    .= SchemaCacheErrorCode00,
     "message" .= ("Could not find a relationship between '" <> parent <> "' and '" <> child <> "' in the schema cache" :: Text),
     "details" .= JSON.Null,
-    "hint"    .= ("Verify that '" <> parent <> "' and '" <> child <> "' exist in the schema '" <> schema <> "' and that there is a foreign key relationship between them. If a new relationship was created, try reloading the schema cache." :: Text)]
+    "hint"    .= ("Verify that '" <> parent <> "' and '" <> child <> "' exist in the schema '" <> schema <> "' and that there is a foreign key relationship between them." :: Text)]
```

Comment:
This seems unrelated, how about leaving this change for another PR?(dedicated to closing https://github.com/PostgREST/postgrest/issues/2569#issuecomment-1334540506)

Reply:
^ easier to review if the change is focused

Reply:
Agree, done.